spec/controllers/users_controller_spec.rb

```diff
@@ -72,28 +83,29 @@
       end
     end
 
-    context "email sending" do
-      it "sends out the email given valid inputs" do
-        post :create, user: Fabricate.attributes_for(
-          :user, email: "example@example.com", full_name: "The Wizard of Id"
+    context "with valid input but invalid card" do
+      let(:charge) {
+        double(
+          :charge,
+          successful?: false, error_message: "Your card was declined"
         )
-        expect(ActionMailer::Base.deliveries.last.to).to eq(["example@example.com"])
+      }
+
+      before do
+        StripeWrapper::Charge.should_receive(:create).and_return(charge)
+        post :create, user: Fabricate.attributes_for(:user),
+                        stripeToken: '12345678'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.